### PR TITLE
[Tool]add repeat runs & add new db create before every case runs

### PR DESCRIPTION
**Pull request by huailiu1122**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: test/lib/sql_annotation.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def receive(func):
         def wrapper(*args, **kwargs):
             """wrapper"""
             name = args[1].name
-            args[0].db = list(args[1].db)
+            args[0].db = args[1].db
```
Comment (andyziye) [inline test/lib/sql_annotation.py:40]: Whether multiple DBS are supported?

File: test/requirements.txt [hunk 2]
```diff
@@ -5,3 +5,4 @@ nose~=1.3.7
 parameterized
 pymysql
 nose_xunitmp
+flaky==3.7.0
```
Comment (andyziye) [inline test/requirements.txt:8]: You need to update this in runner.

File: test/sql/test_storage_volume/R/set_as_default [hunk 3]
```diff
@@ -23,6 +19,10 @@ ALTER STORAGE VOLUME storage_volume_default_1 SET ("aws.s3.region"="us-west-1",
 SET storage_volume_default_1 AS DEFAULT STORAGE VOLUME;
 -- result:
 -- !result
+DESC STORAGE VOLUME 'storage_volume_default_1';
```
Comment (andyziye) [inline test/sql/test_storage_volume/R/set_as_default:22]: What is the reason for the change?

Comment (huailiu1122) [reply test/sql/test_storage_volume/R/set_as_default:22]: case is not stable, STORAGE VOLUME must has one default value, have discussed with case owner

File: test/lib/sql_annotation.py (hunk 1 above)
Comment (andyziye) [reply test/lib/sql_annotation.py:40]: such as db[0], db[1]...

Comment (huailiu1122) [reply test/lib/sql_annotation.py:40]: yes, now DB init is in method init_data, this line actually does't work

Comment (andyziye) [reply test/lib/sql_annotation.py:40]: list(db) is removed, set does not support index?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by andyziye — DISMISSED** [review]

**Review by wanpengfei-git — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)
